ios/FluentUI/List/ListItem.swift

```diff
@@ -436,3 +433,11 @@ public extension ListItem where LeadingContent == EmptyView, TrailingContent ==
         }
     }
 }
+
+public extension ListItem {
+    /// Provide override values for various `ListItem` values.
+    mutating func overrideTokens(_ overrides: [ListItemToken: ControlTokenValue]) -> some View {
```

Comment:
Instead of returning `some View`, can you return either `Self` or `ListItem`? That'll retain the typing a little longer.